src/guide/advanced/triggers.md

```diff
@@ -166,6 +166,22 @@ you have to `Register` it again, with the same name.
 
 <!-- TODO: test if reaches zero needs to re-register. -->
 
+### Domain-scoped Triggers
+
+::: stable
```

Comment:
We should probably rephrase these. 

Instead of saying that it's included in stable, we should instead say that it isn't yet part of the LTS. The component stays the same but the text nees a slight adjustment.

Reply:
Added a note about the LTS. We also mention that LTS supports only un-scoped triggers a few lines above